src/shared/treeview/nodes/errorNode.ts

```diff
@@ -5,13 +5,26 @@
 
 import * as vscode from 'vscode'
 import { AWSTreeNodeBase } from './awsTreeNodeBase'
+import { localize } from '../../../shared/utilities/vsCodeUtils'
 
 // Used as a child node when an exception occurs while querying AWS resources
 export class ErrorNode extends AWSTreeNodeBase {
-    public constructor(public readonly parent: AWSTreeNodeBase, public readonly error: Error, label: string) {
+    public constructor(public readonly parent: AWSTreeNodeBase, public readonly error: Error, label: string, logID: number = -1) {
         super(label, vscode.TreeItemCollapsibleState.None)
+        const commandName: string = localize('AWS.command.viewLogs', 'View AWS Toolkit Logs')
+        const tooltip: string = localize('AWS.explorerNode.error.tooltip', 'Click to view error in Toolkit logs')
```

Comment:
I'm not convinced this is the right pattern. I think there's value in showing the error in the tooltip as we have now...I would consider keeping the existing tooltip and swapping this to a right-click action on the node.

Reply:
Agreed. This was discussed when I demo'd yesterday. The final decision was to make the label: "Failed to load resources (click for details)". The tooltip can then be the error with the message.

Reply:
> (alternately, keep the action onclick but swap the tooltip to the error message)

Ok. I think we should keep it as left-click. Why make it a right-click action?